dataprofiler/tests/labelers/test_data_labelers.py

```diff
@@ -137,11 +137,15 @@ def test_load_data_labeler(self, *mocks):
     def test_load_from_library(self, *mocks):
         data_labeler = dp.DataLabeler.load_from_library("structured_model")
         self.assertIsInstance(data_labeler, BaseDataLabeler)
+        # Testing to ensure _default_model_loc is set correctly
+        self.assertEqual("structured_model", data_labeler._default_model_loc)
 
         data_labeler = dp.DataLabeler.load_from_library(
             "structured_model", trainable=True
         )
         self.assertIsInstance(data_labeler, TrainableDataLabeler)
+        # Testing to ensure _default_model_loc is set correctly
+        self.assertEqual("structured_model", data_labeler._default_model_loc)
```

Comment:
let's have an additional test to test `rase ValueError` when `default_model_loc` is `None`